src/docs/components/dropdown/README.md

```diff
@@ -10,10 +10,7 @@ Import Dropdown, its item and styles from this package. Then use them wherever y
 
 ```JSX
 import React from 'react';
-import { Dropdown, DropdownItem } from '@red-hat-insights/insights-frontent-components/components/Dropdown';
-
-import '@red-hat-insights/insights-frontent-components/components/Dropdown.css';
-// in scss file @import '~@red-hat-insights/insights-frontent-components/components/Dropdown.css';
+import { Dropdown, DropdownItem } from '@redhat-cloud-services/frontend-components';
```

Comment:
We should update this file to reflect PF dropdown https://github.com/RedHatInsights/insights-frontend-storybook/issues/79

Reply:
I'm not going to update it entirely, but I will provide a deprecated warning and a link to the new component.

Reply:
Actually, we've removed this component entirely already, so I'll remove this file.